kong/templates/nginx_kong_stream.lua

```diff
@@ -119,5 +119,15 @@ server {
         Kong.log()
     }
 }
+
+server {
+    listen unix:${{PREFIX}}/stream_config.sock;
+
+    error_log  ${{PROXY_ERROR_LOG}} ${{LOG_LEVEL}};
```

Comment:
The `content_by_lua_block` will be executed in the context of Admin API only. Can we change the error output to `ADMIN_ERROR_LOG`?